_RFCs/0003-tracking-package-versions.md

```diff
@@ -0,0 +1,56 @@
+- Change Name: standardize_tracking_major_package_versions
+- Start Date: 2018-03-27
+- RFC PR: (leave this empty)
+- Core-Plans Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Everytime a new major version of software packaged by Habitat is released (whether stable or an edge release), a new core plan must be made for that version (i.e. when Docker11 is released, we should create core/docker11).
+
+Generic package names (ones which do not specify a version in the name - i.e. core/docker) must always point to the latest **stable** version of the software.  They must not track edge versions of the software.  They must source the plan for the latest stable plan which tracks that software (i.e. if the current stable version of Docker is Docker10, and the edge version is Docker11, core/docker should source core/docker10).
+
+If a user wishes to use an edge version of the software (i.e. Docker11 is the edge version, while Docker10 is the stable version), they can do so by directly depending on core/docker11.
```

Comment:
This might be pedantry, but I 100% don't believe we should be maintaining edge software releases. The overhead on "stable" pieces of software that we're already attempting to repackage and redistribute is enough. Unless we're also committing to maintain edge releases then I feel like we should strip the reference to an optional edge package path for our users. My opinion is we should be tracking stable packages and long term support packages as top level package names in core.

Reply:
My feelings exactly.

Reply:
Perhaps we can add some "if available" language.  I think that the "`core/docker` follows latest stable" makes sense to standardize.  Policy around the overall availability of edge versions seems less valuable.  For some packages, adding an edge version will only add maintenance burden.  For other packages, it might be nice to have edge-versions available sooner if (1) there is a community of downstream packages that would otherwise have to rely on something outside of core or (2) there is an active maintainer who thinks that having the package in core is the best way to resolve various downstream issues before the package becomes stable.

Reply:
I could agree with the process you've outlined @stevendanna so long as it's codified in the RFC. I think blanket adding verbiage around edge packages is a no-go for me.